airflow/providers/microsoft/azure/log/wasb_task_handler.py

```diff
@@ -92,7 +92,7 @@ def hook(self):
             )
             return None
 
-    def set_context(self, ti) -> None:
+    def set_context(self, ti):
```

Comment:
You need to add `identifier` to all the task handlers, no? Do I miss something? :thinking:

Reply:
Yes, you're right. I thought to add a quick basic PR here to add the feature in core and then subsequent PR to add those needed support in the providers. The initial plan was to get this bare minimum feature support in 2.7 and then to add the provider supports in subsequent PR as they can be released subsequently once this support is available in core.